Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mhart/alpine-node:13.12.0
```

Comment:
What will be the workflow of running the codebase on local, after these changes? Can you add a basic readme with the steps?

Reply:
May be we can merge this PR for now. Once we setup the Docker deployment for server, we can add the full documentation.

As of now, the docker set up only works for local development.